src/PlaywrightSharp.Abstractions/IBrowser.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace PlaywrightSharp
+{
+    /// <summary>
+    /// A Browser is created when Playwright connects to a browser instance
+    /// TODO: Complete when we have more APIs to refer to
+    /// </summary>
+    public interface IBrowser
```

Comment:
consider extending `IAsyncDisposable`

Reply:
These are some early scaffoldings.  On the other hand, I don't think we are going to go to .NET Standard 2.1. That will leave the .NET Framework out, and I don't think we want that.